site/docs/pages/token/token-image.mdx

```diff
@@ -5,7 +5,7 @@ import App from '../../components/App';
 
 The `TokenImage` component is an image that crops token image to a circle with an adjustable size.
 
-With `token` props has no image, render partial token symbol and deterministic dark color.
+When `token` props has no image, render partial token symbol and deterministic dark color.
```

Comment:
Good catch - I don't think we need this, as it's covered below.  Let's remove it
```suggestion
```